### ISSUE TITLE
Support Aix

### ISSUE THREAD

**Issue by hmmftg** [creack/pty#139]
Hi when I cross compile gotty module for aix like this:

    `env GOOS=aix GOARCH=ppc64 make`

I got this error:  
     `# github.com/kr/pty`
     `vendor/github.com/kr/pty/ioctl.go:6:29: undefined: syscall.SYS_IOCTL`
     `vendor/github.com/kr/pty/util.go:26:3: undefined: syscall.SYS_IOCTL`

can you please add support for aix? it's similar to solaris and in sys/unix, ioctl is similar to solaris

thanks

**Comment by creack** [comment]
Indeed, we do not support this os/arch yet. Unfortunately, I don't have any way to try. Would you be able to provide a live VM or SSH session so I could test?

**Comment by creack** [comment]
@hmmftg is #144 enough or do you need actual support? Without any knowledge or mean of testing, I can't do much more at the time.

**Comment by creack** [comment]
Note that you will need to update the dependency from github.com/kr/pty to github.com/creack/pty and use v1.1.18 (https://github.com/creack/pty/releases/tag/v1.1.18) to get the fix.

**Comment by hmmftg** [comment]
> do you need actual suppo

Hi 
Sorry for my late response
I want to use a package called cloudshell, your commit let me cross compile and build new cloudshell for our remote aix(unreachable from internet)
but it gaves me this error: handler_websocket.go xtermjs.GetHandler.func1 failed to start tty: unsupported

So yes, we need actual support for this
how can I be help on testing or any means for this?

as far I tried some go packages and we have this issue in viper, fsnotify and etc, most of times we change the *_solaris.go and add +aix on build command
